kong/plugins/acl/schema.lua

```diff
@@ -4,7 +4,8 @@ return {
   no_consumer = true,
   fields = {
     whitelist = { type = "array" },
-    blacklist = { type = "array" }
+    blacklist = { type = "array" },
+    set_groups_header = { type = "boolean", default = true }
```

Comment:
The default for this value should probably respect the already existing behaviour, that is to _include_ the groups header. Thus, `default = false`. Now, to semantically align with the authentication plugins, which provide a `hide_credentials` option (whose default is `false`), this option should probably be updated to `hide_groups_header = { type = "boolean", default = false }`. Thoughts?

Reply:
And then just do this right?
```
  if not conf.hide_groups_header then
    set_header(constants.HEADERS.CONSUMER_GROUPS, to_be_blocked)
  end
```
This works fine for me 👍 , whatever Kong prefers honestly. Glad to see yall are cool with adding this functionality.

Reply:
@jeremyjpj0916 Yes, exactly.